src/nrnpython/nrnpy_nrn.h

```diff
@@ -166,6 +166,18 @@ static PyType_Spec nrnpy_RangeType_spec = {
     nrnpy_RangeType_slots,
 };
 
+static PyType_Slot nrnpy_OpaquePointerType_slots[] = {
+    {Py_tp_doc, (void*) "Opaque pointer."},
+    {0, 0},
```

Comment:
Ahah better with `{0, nullptr}`

Reply:
True, but consistency requires `{0, 0}` :)

Reply:
Tomorrow I'm running clang-tidy to modernize these `0`-pointers.